monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md

```diff
@@ -0,0 +1,129 @@
+# Breaking Monero 11: «Почтовый» эпизод  
+*06/09/19*  
+_**В этом эпизоде мы отвечаем на вопросы зрителей.**_  
+
+**Breaking Monero Эпизод 11 - «Почтовый» эпизод**  
+
+https://youtu.be/q1gdYWsvXDU  
+В этом эпизоде мы ответим на несколько вопросов, полученных от наших зрителей. Книги и источники:
+
+[Введение в математическую криптографию](https://www.goodreads.com/book/show/3884171-an-introduction-to-mathematical-cryptography)  
+[Введение в современную криптографию](https://www.goodreads.com/book/show/24387035-introduction-to-modern-cryptography-second-edition)  
+[Интенсивное введение в криптографию](https://intensecrypto.org/public/index.html)  
+[Monero Konferenco](https://monerokon.com/)  
+
+[Все эпизоды Breaking Monero](https://www.monerooutreach.org/breaking-monero/)
+
+_**Стенограмма эпизода:**_
+
+_**Джастин:**_ Добро пожаловать на одиннадцатый эпизод Breaking Monero. Это специальный эпизод, в рамках которого мы ответим на вопросы, присланные нам на почту людьми в связи с прошлыми эпизодами. Мы — это я, Джастин, Брендон и Саранг. На самом деле, вопросов довольно много, и мы надеемся, что у нас получится восполнить некоторые пробелы или разъяснить некоторые серии Breaking Monero, или же коснуться проблем, которые, по вашему мнению, имеют отношение к контрибьюторам Исследовательской лаборатории Monero. Итак, Брендон, Саранг, начнём, пожалуй.
+
+_**Джастин:**_ В первом полученном нами вопросе, касающимся темы, затронутой во втором эпизоде, говорится о том, что во время одного из эпизодов Monero Talk Эндрю Поэлстра высказал идею, согласно которой Monero может либо обеспечивать совершенную анонимность, используя свои механизмы сокрытия средств, либо твёрдую гарантию невозможности создания таких средств. Не мог бы ты сравнить идеальное связывание с идеальным сокрытием, Саранг?
```

Comment:
Hey @v1docq47,

Will leave you comments on the paragraphs I've seen that  are missing links. This one is one of them, you will just need to add them >:).

- [episode 2](https://www.monerooutreach.org/breaking-monero/ring-signatures.php)
- [Monero Talk](https://www.monerotalk.live/)